keyboards/planck/keymaps/lindgrenj6/keymap.c

```diff
@@ -0,0 +1,227 @@
+    _______, _______, _______, _______, _______, _______, KC_PIPE, KC_LCBR,           KC_RCBR,           KC_PLUS,        KC_EQUAL,       LT(KC_PGDN, KC_END),
+    _______, _______, _______, _______, _______, _______, _______, KC_LPRN,           KC_RPRN,           _______,        KC_BSLS,        KC_DELETE,
+    _______, _______, _______, _______, _______, _______, _______, _______,           C(S(KC_LEFT)),     C(KC_MINUS),    C(S(KC_PLUS)), C(S(KC_RIGHT))
+  ),
+
+  [_RAISE] = LAYOUT_planck_grid(
+    KC_GRAVE, KC_1,   KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    LGUI(KC_PGUP),
+    _______,  KC_F1,  KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F11,  _______, _______, _______, _______, LGUI(KC_PGDN),
+    _______,  KC_F6,  KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F12,  _______, _______, _______, KC_BSLS, _______,
+    _______, _______, _______, _______, _______, KC_PSCR, KC_PSCR, _______, _______, _______, _______, KC_MPLY
+  ),
+
+  [_ADJUST] = LAYOUT_planck_grid(
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, AU_ON,   AU_OFF,  AU_TOGG, RGB_SPI, _______, RGB_TOG, RGB_VAI, RGB_HUI, KC_BRIGHTNESS_UP,   QK_BOOT,
+    _______, _______, MU_ON,   MU_OFF,  MU_TOGG, RGB_SPD, _______, RGB_MOD, RGB_VAD, RGB_HUD, KC_BRIGHTNESS_DOWN, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______
+  ),
+
+  [_NUMPAD] = LAYOUT_planck_grid(
+    _______, _______, _______, _______, _______, _______, KC_CALCULATOR, KC_KP_7, KC_KP_8, KC_KP_9,   KC_KP_MINUS, KC_KP_SLASH,
+    _______, _______, _______, _______, _______, _______, _______,       KC_KP_4, KC_KP_5, KC_KP_6,   KC_KP_PLUS,  KC_KP_ASTERISK,
+    _______, _______, _______, _______, _______, _______, _______,       KC_KP_1, KC_KP_2, KC_KP_3,   KC_KP_ENTER, _______,
+    _______, _______, _______, _______, _______, _______, _______,       KC_KP_0, _______, KC_KP_DOT, KC_KP_ENTER, _______
+  ),
+
+  [_EXTRA] = LAYOUT_planck_grid(
+    G(KC_L), KC_SLEP, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, KC_LEFT,        KC_DOWN,        KC_UP,          KC_RIGHT,       _______, _______,
+    _______, _______, KC_LCTL,       _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, KC_AUDIO_MUTE, KC_AUDIO_VOL_DOWN, KC_AUDIO_VOL_UP, LT(_EXTRA, KC_MPLY)
+  ),
+
+};
+
+#ifdef AUDIO_ENABLE
+  float plover_song[][2]     = SONG(PLOVER_SOUND);
+  float plover_gb_song[][2]  = SONG(PLOVER_GOODBYE_SOUND);
+#endif
+
+layer_state_t layer_state_set_user(layer_state_t state) {
+  return update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
+}
+
```

Comment:
The core [tri layer feature](https://docs.qmk.fm/#/feature_tri_layer) should be used instead.
```suggestion
```